File: Plugins/Aspose_Slides_Java_for_PHP/src/aspose/slides/WorkingWithPresentation/PptToPptx.php
<?php

namespace Aspose\Slides\WorkingWithPresentation;

use com\aspose\slides\Presentation as Presentation;
use com\aspose\slides\SaveFormat as SaveFormat;

class PptToPptx {

    public static function run($dataDir=null)
    {
        # Instantiate a Presentation object that represents a PPTX file
        $pres = new Presentation($dataDir . "Presentation1.ppt");

        # Saving the PPTX presentation to PPTX format
        $save_format = new SaveFormat();
        $pres->save($dataDir . "Aspose.pptx", $save_format->Pptx);

        print "Document has been converted, please check the output file.";
    }

}

File: Plugins/Aspose-Slides-Java-for-Jython/asposeslides/WorkingWithPresentation/PptToPptx.py
from asposeslides import Settings
from com.aspose.slides import Presentation
from com.aspose.slides import SaveFormat
from com.aspose.slides import XpsOptions

class PptToPptx:

    def __init__(self):
                        
         
        dataDir = Settings.dataDir + 'WorkingWithPresentation/PptToPptx/'
        
        # Instantiate a Presentation object that represents a PPTX file
        pres = Presentation(dataDir + "Presentation1.ppt")

        # Saving the PPTX presentation to PPTX format
        save_format = SaveFormat
        pres.save(dataDir + "Aspose.pptx", save_format.Pptx)

        print "Document has been converted, please check the output file."


if __name__ == '__main__':        
    PptToPptx()